src/utils/validation.ts

```diff
@@ -151,7 +151,7 @@ export function validateIRelayerMakerOrder(order: IRelayerMakerOrder) {
   if (expiry !== 2209006800) {
     return "expiry incorrect";
   }
-  if (apiExpiry >= Date.now() / 1000) {
+  if (apiExpiry <= Date.now() / 1000) {
```

Comment:
Previously, in `if (!(parseInt(expiry, 10) >= Date.now() / 1000)) {`, this has been reversed on a purpose to not allow `NaN`, which are numbers, to pass.

`if (!(apiExpiry >= Date.now() / 1000))` is a safer check as it doesn't allow `NaN` values.

Also, there seems to be check for that value to be a number at all? Not sure if that's needed.

Reply:
hey @ChALkeR sure we can change this. I'll enforce it to be a number too.